src/components/Common/UI/Switch/Switch.module.scss

```diff
@@ -0,0 +1,92 @@
+.root {
+  :global(.react-aria-Switch) {
+    --selected-color: var(--g-colors-gray-1000);
+    --selected-color-pressed: var(--g-colors-gray-900);
+
+    position: relative;
+
+    font-size: toRem(15);
+    color: var(--g-typography-textColor);
+    cursor: pointer;
+    forced-color-adjust: none;
+
+    .body {
+      display: flex;
+      align-items: start;
+      gap: 0.571rem;
```

Comment:
Nit: i know these are copy/pasted styles but we should probably circle back to fix styles like this that are rendering subpixel (since diff browsers handle them differently). Ex. imo this should be one of the following

```
toRem(8) //.5rem
toRem(9) // .5625rem
0.5rem
0.5625rem
```

Reply:
Will be resolved in next iteration.